src/connection/server_support.c

```diff
@@ -1390,6 +1390,11 @@ css_init (THREAD_ENTRY * thread_p, const char *server_name, int port_id)
 	  css_setup_server_loop ();
 	}
     }
+  else
+    {
+      er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ERR_CSS_ERROR_DURING_SERVER_CONNECT, 1, message_to_master.c_str () + 1);
+      return ERR_CSS_ERROR_DURING_SERVER_CONNECT;
```

Comment:
- Let it fall through to do the whole shutdown sequence.
- Use `server_name` as argument.
```suggestion
      er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ERR_CSS_ERROR_DURING_SERVER_CONNECT, 1, server_name);
      status = ERR_CSS_ERROR_DURING_SERVER_CONNECT;
```